pkgs/os-specific/linux/fwts/default.nix

```
@@ -20,6 +20,12 @@ stdenv.mkDerivation rec {
       --replace "/usr/bin/lspci"      "${pciutils}/bin/lspci" \
       --replace "/usr/sbin/dmidecode" "${dmidecode}/bin/dmidecode" \
       --replace "/usr/bin/iasl"       "${acpica-tools}/bin/iasl"
+
+    substituteInPlace src/lib/src/fwts_devicetree.c \
+      --replace "dtc -I" "${dtc}/bin/dtc -I"
+
+    substituteInPlace src/devicetree/dt_base/dt_base.c \
+      --replace "dtc -I" "${dtc}/bin/dtc -I"
```

```suggestion
    substituteInPlace src/lib/src/fwts_devicetree.c \
                      src/devicetree/dt_base/dt_base.c \
      --replace "dtc -I" "${dtc}/bin/dtc -I"
```

`substituteInPlace` accepts multiple files as parameters.

---

TIL. Thanks. Fixed in latest push